binaries/cli/src/lib.rs

```diff
@@ -1,38 +1,13 @@
 use colored::Colorize;
-use communication_layer_request_reply::{RequestReplyLayer, TcpLayer, TcpRequestReplyConnection};
-use dora_coordinator::Event;
-use dora_core::{
-    descriptor::{source_is_url, Descriptor, DescriptorExt},
-    topics::{
-        DORA_COORDINATOR_PORT_CONTROL_DEFAULT, DORA_COORDINATOR_PORT_DEFAULT,
-        DORA_DAEMON_LOCAL_LISTEN_PORT_DEFAULT,
-    },
-};
-use dora_daemon::{Daemon, LogDestination};
-use dora_download::download_file;
-use dora_message::{
-    cli_to_coordinator::ControlRequest,
-    coordinator_to_cli::{ControlRequestReply, DataflowList, DataflowResult, DataflowStatus},
-};
-#[cfg(feature = "tracing")]
-use dora_tracing::TracingBuilder;
-use duration_str::parse;
-use eyre::{bail, Context};
-use formatting::FormatDataflowError;
-use std::{env::current_dir, io::Write, net::SocketAddr};
+use command::Executable;
 use std::{
     net::{IpAddr, Ipv4Addr},
     path::PathBuf,
-    time::Duration,
 };
-use tabwriter::TabWriter;
-use tokio::runtime::Builder;
-use tracing::level_filters::LevelFilter;
-use uuid::Uuid;
 
-pub mod command;
+mod command;
```

Comment:
I think we need to keep the command module public because the Python node API crate needs to be able to invoke the `run` function.